src/languages/en.js

```diff
@@ -246,6 +246,11 @@ export default {
         newFaceEnterMagicCode: ({login}) => `It's always great to see a new face around here! Please enter the magic code sent to ${login}. It should arrive within a minute or two.`,
         welcomeEnterMagicCode: ({login}) => `Please enter the magic code sent to ${login}. It should arrive within a minute or two.`,
     },
+    DownloadAppModal: {
```

Comment:
I dont think other keys are capitalized
```suggestion
    downloadAppModal: {
```